# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/guile/guile-gcc4.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./libguile/guile.c.vanilla	2005-12-12 23:23:05.000000000 +0100
+++ ./libguile/guile.c	2005-12-12 23:23:56.000000000 +0100
@@ -88,8 +88,9 @@
 {
 #ifdef DYNAMIC_LINKING
   /* libtool automagically inserts this variable into your executable... */
-  extern const scm_lt_dlsymlist lt_preloaded_symbols[];
-  scm_lt_dlpreload_default (lt_preloaded_symbols);
+// commenting the next two lines out for gcc4 as suggested on the guile ml
+//  extern const scm_lt_dlsymlist lt_preloaded_symbols[];
+//  scm_lt_dlpreload_default (lt_preloaded_symbols);
 #endif
   scm_boot_guile (argc, argv, inner_main, 0);
   return 0; /* never reached */
